src/Generator/ClassFinder.php

```diff
@@ -0,0 +1,213 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\ApiDocumenter\Generator;
+
+use DateTime as BaseDatetime;
+use DateTimeInterface;
+use EoneoPay\Utils\DateTime;
+use EoneoPay\Utils\UtcDateTime;
+use Symfony\Component\PropertyInfo\Extractor\PhpDocExtractor;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractor;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractorInterface;
+use Symfony\Component\PropertyInfo\Type;
+
+final class ClassFinder
```

Comment:
I dont know if this is an efficient or maintainable approach and would appreciate feedback.